### ISSUE TITLE
Demo page returns 404

### ISSUE THREAD

**Issue by dboydor** [htapal/angular-packery#1]
http://passy.github.io/angular-packery doesn't seem to exist anymore

Do you have another location to look at it?

**Comment by htapal** [comment]
Yep you can find it at https://github.com/htapal/angular-packery

**Comment by dboydor** [comment]
Well, I was looking for a page already hosted that I could click on, but thanks anyway.
